ebpf/symtab/kallsyms.go

```diff
@@ -94,5 +95,9 @@ func NewKallsymsFromData(kallsyms []byte) (*SymbolTab, error) {
 	if allZeros {
 		return NewSymbolTab(nil), nil
 	}
+	// kallsyms maybe unsorted when bpf/modules are loaded from userspace after kernel boot.
+	sort.Slice(syms, func(i, j int) bool {
```

Comment:
nit: It would be nice to add a test, but it is is not required

Reply:
PR Updated. I add 2 unordered symbols in testdata. I tested it and it will fail without this patch and success with this patch.